starter_code/index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <link rel="stylesheet" href="css/style.css" media="screen" title="no title" charset="utf-8">
+
+    <title>Relaxr</title>
+  </head>
+  <body>
+    <h1>Relaxr</h1>
+    <h2>Get piece of mind with a single tap</h2>
+    <input type="button" name="name" value="Get it Now">
+
+    <h2>Benefits</h2>
+    <p>
+      The perfect personal assistant. Relaxr does your job for you so you can enjoy life as it is meant to be.
+    </p>
+    <ul>
+      <li>Scheule meetings for you</li>
+      <li>Excel automation to complete your work for you</li>
+      <li>Responds to emails on your behalf</li>
+      <li>Does all your work for you with our revolutionary AI technology</li>
+    </ul>
+    <h3>"Relaxr changed my life. I've been able to travel the world, spend limited time working and my boss keeps thanking me for crushing work."
+    <h3>-Amanda, Intuit</h3>
+
+    <input type="button" name="name" value="Sign Up Now">
+
+    <h3>Relaxr</h3>
+
+    <img src="images/twitter.png" alt="twitter" />
```

Comment:
These should probably be wrapped in `<a>`. The intuition is for users to visit Relaxr's social media pages